jetpack.php

```diff
@@ -235,6 +235,7 @@ function jetpack_admin_missing_autoloader() { ?>
 require_once( JETPACK__PLUGIN_DIR . 'class.jetpack-connection-banner.php'  );
 require_once( JETPACK__PLUGIN_DIR . 'class.jetpack-plan.php'          );
 
+Jetpack::initialize_sync();
```

Comment:
This feels like an anti-pattern. I don't want a whole glut of similar initialisation happening when the plugin file is loaded. Hopefully almost NO code gets loaded at first, just a few clean hooks that load everything else on demand.